backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -956,6 +992,19 @@ private CodeableConcept createLoincConcept(String codingCode, String codingDispl
     return concept;
   }
 
+  private CodeableConcept createSimpleReportConcept(
+      String codingCode, String codingDisplay, String text) {
+    CodeableConcept concept = new CodeableConcept().setText(text);
+
+    concept
+        .addCoding()
+        .setSystem(SIMPLE_REPORT_CODE_SYSTEM)
```

Comment:
Is there any additional context or discussion about any implications of what it means for us to send this data using a newly created SimpleReport code system? Do we need to create documentation links about this for ReportStream or STLTs?

Reply:
In the schema that SR requested they did specifically request this code system. Here's a snipet of the json that they provided here:

```
    "status" : "final",
    "code" : {
      "coding" : [
        {
          "system" : "http://simplereport.gov/",
          "code" : "SR0001",
          "display" : "What is the gender of their sexual partners"
        }
      ],
      "text" : "What is the gender of their sexual partners"
    },
    "subject" : {
      "reference" : "Patient/123"
    }
```

Reply:
The whole schema that they defined is [written up in the issue too](https://github.com/CDCgov/prime-simplereport/issues/7185).  In the issue it also notes that we should RS before merging so they are aware that the new coding system is going live.

@mehansen Can you confirm that that's why we not to notify them beforehand?